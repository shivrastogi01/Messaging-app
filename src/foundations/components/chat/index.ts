export * from "./text-message/text-message";
export * from "./image-message/image-message";
export * from "./link-message/link-message";
export * from "./giphy-message/giphy-message";
export * from "./avatar/avatar";
