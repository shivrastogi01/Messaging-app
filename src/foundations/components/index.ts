export * from "./presentation";
export * from "./layout";
