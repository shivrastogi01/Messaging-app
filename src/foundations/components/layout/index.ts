export * from "./styled-box/styled-box";
export * from "./scroll-view/scroll-view";
export * from "./flex-row/flex-row";
export * from "./flex-column/flex-column";
export * from "./modal/modal";
export * from "./drawer/drawer";
export * from "./list-item/list-item";
export * from "./dropdown/dropdown";
