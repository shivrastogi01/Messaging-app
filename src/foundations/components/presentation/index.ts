export * from "./heading/heading";
export * from "./label/label";
export * from "./title/title";
export * from "./icon/icon";
export * from "./button/button";
export * from "./input/input";
export * from "./textarea/textarea";
