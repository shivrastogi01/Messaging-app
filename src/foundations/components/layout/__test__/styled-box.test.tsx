import React from "react";
import renderWithTheme from "main/testing/render-with-theme";
import { StyledBox } from "foundations/components/layout";

describe("StyledBox", () => {
  test("default variant matches the snapshot", () => {
    const { container } = renderWithTheme(<StyledBox>Scroll View</StyledBox>);
    expect(container.firstChild).toMatchSnapshot();
  });

  test("with some props matches the snapshot", () => {
    const { container } = renderWithTheme(
      <StyledBox height={200} width={200} margin={1} padding={1}>
        Scroll View
      </StyledBox>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
