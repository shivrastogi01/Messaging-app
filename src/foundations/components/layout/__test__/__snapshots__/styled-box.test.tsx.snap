// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledBox default variant matches the snapshot 1`] = `
<div
  class=""
>
  Scroll View
</div>
`;

exports[`StyledBox with some props matches the snapshot 1`] = `
.c0 {
  margin: 10px;
  padding: 10px;
  height: 200px;
  width: 200px;
}

<div
  class="c0"
  height="200"
  width="200"
>
  Scroll View
</div>
`;
